src/TeamleaderDotNet/Utils/UnixTimestampExtensions.cs

```diff
@@ -44,7 +44,7 @@ public static DateTime UnixTimeToDateTime(this int unixtime)
         /// <returns>Returns a DateTime object that represents value of the Unix time.</returns>
         public static DateTime UnixTimeToDateTime(this string unixtime)
         {
-            return UnixTimeToDateTime(long.Parse(unixtime.ToString()));
+            return !string.IsNullOrEmpty(unixtime) ? UnixTimeToDateTime(long.Parse(unixtime)) : DateTime.MinValue;

```

Comment:
Wouldn't it be nicer if we return NULL instead of DateTime.MinValue (or make another extension method so that we can support both cases). For me it feels more natural that something without date is NULL instead of DateTime.MinValue. This would have the effect though that some of properties will also be changed in the future to nullable types - so not that great for backwards compatibility.